### PR TITLE
arrow.coffee typo correction

### LINKED ISSUE #4998: typo in arrow.coffee
An `@` is missing on line 22, before `set_data()`

### PR DESCRIPTION
**Pull request by Maggie-M**
issues: fixes #4998

### CODE REVIEW

File: bokehjs/src/coffee/models/annotations/arrow.coffee [hunk 1]
```diff
@@ -19,7 +19,7 @@ class ArrowView extends Annotation.View
   bind_bokeh_events: () ->
     @listenTo(@model, 'change', @plot_view.request_render)
     @listenTo(@mget('source'), 'change', () ->
-      set_data()
+      @set_data()
```
Comment (mattpap) [inline bokehjs/src/coffee/models/annotations/arrow.coffee]: Is this code ever executed? Maybe this listener can be removed altogether?

Comment (bryevdv) [inline bokehjs/src/coffee/models/annotations/arrow.coffee]: Yes, it came up in a mailing list issue when I ran a users code

Comment (mattpap) [inline bokehjs/src/coffee/models/annotations/arrow.coffee]: OK.

**Comment by canavandl** [comment]
LGTM

**Pull request merged**